### PR TITLE
rep_sample_n() is now in the infer package

### PR DESCRIPTION
**Pull request by davidkane9**
Love the book and using this chapter this week in my class, hence my close reading of it.

### CODE REVIEW

File: 07-sampling.Rmd [hunk 1]
```diff
@@ -200,7 +200,7 @@ bowl
...
 
-Now that we have a virtual analog of our bowl, we now need a virtual analog to the shovel seen in Figure \@ref(fig:sampling-exercise-2) to generate virtual samples of 50 balls. We're going to use the `rep_sample_n()` function included in the `moderndive` package. This function allows us to take `rep`eated, or `rep`licated, `samples` of size `n`. 
+Now that we have a virtual analog of our bowl, we now need a virtual analog to the shovel seen in Figure \@ref(fig:sampling-exercise-2) to generate virtual samples of 50 balls. We're going to use the `rep_sample_n()` function included in the `infer` package. This function allows us to take `rep`eated, or `rep`licated, `samples` of size `n`.
```
Comment (rudeboybert) [inline 07-sampling.Rmd]: Hey @ismayc. At this point I don't remember why we imported `infer::rep_sample_n()` into `moderndive`. Should use `rep_sample_n()` via `moderndive`, or via `infer`? If we do that latter, then we'll have to include a message to the effect of "Let's load infer, but we'll discuss it in the next Chapter". Thoughts?

Comment (ismayc) [reply 07-sampling.Rmd]: It was exactly for this reason. We don’t want to confuse students by introducing a whole new package that we won’t discuss until a later chapter. I recommend we keep it as `ModernDive` to continue building on that throughout the chapter.

Comment (rudeboybert) [reply 07-sampling.Rmd]: @davidkane9 could you revert the earlier change of which pkg we use? We'll then approve. Thanks for going over this with a fine toothed-comb

**Comment by ismayc** [comment]
The typos in Chapter 7 have been included in #218. Thus, I'm going to close this PR.

File: 07-sampling.Rmd (hunk 1 above)
Comment (davidkane9) [reply 07-sampling.Rmd]: Sorry. Missed this. But looks like you have fixed the other issue.

Comment (ismayc) [reply 07-sampling.Rmd]: No problem! Thanks much.

**Pull request closed without merging**